conan/qasm/conanfile.py

```diff
@@ -35,11 +34,7 @@ class QasmConan(ConanFile):
     description = "A flex/bison parser for OpenQASM v3. A part of the Quantum Engine project."
 
     def source(self):
-        token = os.environ.get("GITHUB_PAT")
-        if token is not None:
-            self.run(f"git clone https://{token}@github.com/openqasm/qe-qasm.git .")
-        else:
-            self.run("git clone git@github.com:openqasm/qe-qasm.git .")
+        self.run("git clone git@github.com:openqasm/qe-qasm.git .")
```

Comment:
I believe this is going to fail again unless you change it to:
```suggestion
        self.run("git clone https://github.com/openqasm/qe-qasm.git .")
```

Reply:
Fixed in [d8539bf](https://github.com/openqasm/qe-compiler/pull/215/commits/d8539bf1267279a3efbcc656b010b21b7ed4ffa7)